ci.sh

```diff
@@ -2,4 +2,5 @@
 
 /usr/local/nginx/sbin/nginx &
 sleep 1
-curl http://127.0.0.1/fixture.car?bytes=0:1024
+curl http://127.0.0.1/
+cat /var/log/nginx/error.log
```

Comment:
maybe there's nothing in the error log?

Reply:
Mmm `debug` statement at the end of the line means it should print the whole output... That's what it does in your VM.

Reply:
i don't think nginx writes it's error log synchronously, so i suspect that the output hasn't been written to the error.log file at the point you read it

Reply:
This is super weird, I made it wait 1s and still nothing. It's like the module isn't even running, even if it was crashing it would say the server sent an empty response...